pkgs/development/libraries/hunspell/dictionaries.nix

```
@@ -856,4 +856,28 @@ rec {
     readmeFile = "README_hr_HR.txt";
     license = with lib.licenses; [ gpl2Only lgpl21Only mpl11 ];
   };
+
+  /* TOKI PONA */
+
+  tok = mkDict rec {
+    pname = "hunspell-dict-tok";
+    version = "20220829";
+    dictFileName = "tok";
+    readmeFile = "README.en.adoc";
+
+    src = fetchzip {
+      url = "https://github.com/somasis/${pname}/releases/download/${version}/${pname}-${version}.tar.gz";
+      sha256 = "sha256-RiAODKXPUeIcf8IFcU6Tacehq5S8GYuPTuxEiN2CXD0=";
+    };
+
+    dontBuild = true;
+
+    meta = with lib; {
+      description = "Hunspell dictionary for Toki Pona";
+      homepage = "https://github.com/somasis/${pname}";
```

```suggestion
      homepage = "https://github.com/somasis/hunspell-dict-tok";
```
Now I can easily click on the link while reviewing